README.md

```diff
@@ -1,46 +1,95 @@
+    - ASP.NET and web development
+- Install [SQL Server 2017 Developer Edition](https://go.microsoft.com/fwlink/?linkid=853016)
+- Install [SQL Management Studio](https://docs.microsoft.com/en-us/sql/ssms/download-sql-server-management-studio-ssms)
+- Install [Azure Storage Emulator](https://go.microsoft.com/fwlink/?linkid=717179&clcid=0x409) (Make sure you are on atleast v5.3)
+- Install [Azure Storage Explorer](http://storageexplorer.com/) 
+- Administrator Access
+
+##### Alternatives (caveat emptor)
+
+- [Visual Studio Code](https://code.visualstudio.com/download)
 
 #### Setup
-- Obtain the appropriate JSON configuration file for this application.
-- Create a Configuration table in your storage account.
-- Add a row to the Configuration table with fields: PartitionKey: LOCAL, RowKey: SFA.DAS.QnA.Api_1.0, Data: {The contents of the JSON Configuration file}.
-
-##### Open the solution
-- Navigate to src/SFA.DAS.QnA.Api/
-- run `dotnet build`
-- run `dotnet test`
-- run `dotnet run`
 
-#### Swagger Documentation
-- https://localhost:5555/swagger/index.html
+- Clone this repository
+- Open Visual Studio as an administrator
+
+##### Publish Database
+
+- Build the solution SFA.DAS.QnA.sln
+- Either use Visual Studio's `Publish Database` tool to publish the database project SFA.DAS.QnA.Database to name {{database name}} on {{local instance name}}
+- To include the latest question sets when publishing a database to your local SQL Server, you need to ensure that `ProjectPath` variable contains the full path in the format  `{{drive}}:\{{project-folders}}\das-qna-api\src\SFA.DAS.QnA.Database\` 
+  - eg. `C:\Source\Repos\SFA\das-qna-api\src\SFA.DAS.QnA.Database\`
+
+  or
+
+- Create a database manually named {{database name}} on {{local instance name}} and run each of the `.sql` scripts in the SFA.DAS.QnA.Database project.
 
-#### QnA Config Tool
-- https://qna-config.apprenticeships.education.gov.uk/
+##### Config
 
-####  SFA.DAS.AssessorService.QnA
-Web API project exposing HTTP REST end points
-- Refer to Swagger documentation
+- Get the das-qna-api configuration json file from [das-employer-config](https://github.com/SkillsFundingAgency/das-employer-config/blob/master/das-qna-api/SFA.DAS.QnA.Api.json); which is a non-public repository.
+- Create a Configuration table in your (Development) local Azure Storage account.
+- Add a row to the Configuration table with fields: PartitionKey: LOCAL, RowKey: SFA.DAS.QnA.API_1.0, Data: {{The contents of the local config json file}}.
+- Update Configuration SFA.DAS.QnA.API_1.0, Data { "SqlConnectionstring":"Server={{local instance name}};Initial Catalog={{database name}};Trusted_Connection=True;" }
 
-#### SFA.DAS.QnA.Api.Types
-Common types to interact with the QnA Api
-- Published to Nuget: https://www.nuget.org/packages/SFA.DAS.QnA.Api.Types
+##### Run the solution
 
-#### SFA.DAS.QnA.Api.Views
-Common views to display questions
-- Published to Nuget: https://www.nuget.org/packages/SFA.DAS.QnA.Api.Views
```

Comment:
I'd keep these in for information so that people know some stuff is published to NuGet

Reply:
When you view this as a Markdown file - all external items are at the top with badges - these have Nuget build badges giving their latest version and a link to the nuget package.